GOVERNANCE.md

```diff
@@ -25,4 +26,25 @@ Contributors are encouraged to ask for collaborator access once you have had a f
 
 ## Process for releasing a new version of Wikimate
 
-[TODO]
+A PR should be created with all relevant changes to update the repository for the upcoming release.
+(See [#81](https://github.com/hamstar/Wikimate/pull/81) for an example.)
+It should apply the following actions:
+
+1. Change the "Unreleased" heading in the CHANGELOG.md file
```

Comment:
[Discussion](https://github.com/hamstar/Wikimate/pull/45#issuecomment-252443635) led to issue #48, and the resulting commits were approved without further discussion it seems. So I guess our agreement was more implicit than I remembered.

Does adhering to a standard mean that we have no creative freedom to deviate in how we word something? [Precedent](https://gcc.gnu.org/wiki/GFortran/News#gfortran_7_.28current_development_version.29) [does](http://docs.nagvis.org/1.3/en_US/changelog.html) [exist](http://pyhrf.org/changelog.html), although I've not found many more examples.
Anyway, I like "current development" better because it has a more positive and active connotation. Many words starting with "un" feel less so.

How exactly do you mean "fits better with the other headings"?